protobuf/signalling.proto

```diff
@@ -89,7 +89,8 @@ message DecoyList {
 // State transitions of the client
 enum C2S_Transition {
     C2S_NO_CHANGE = 0;
-    // C2S_SESSION_INIT = 1; // Currently MSG_INIT is NOT protobuf
+    C2S_SESSION_INIT = 1; // connect me to squid
+    C2S_SESSION_COVERT_INIT = 11; // connect me to provided covert
```

Comment:
11?

Reply:
11!

I'd be happy to use something else if you have suggestions.
Didn't want to use 5, because 5 is already used in `S2C_Transition`, and I wanted `S2C_Transition` and `C2S_Transition`s values to correspond to each other.